README.md

```diff
@@ -1,75 +1,62 @@
+# [redux-simple-router](#docs)
 
-# redux-simple-router
+[![npm version](https://img.shields.io/npm/v/redux-simple-router.svg?style=flat-square)](https://www.npmjs.com/package/redux-simple-router) [![npm downloads](https://img.shields.io/npm/dm/redux-simple-router.svg?style=flat-square)](https://www.npmjs.com/package/redux-simple-router)
 
-[![npm version](https://img.shields.io/npm/v/redux-simple-router.svg?style=flat-square)](https://www.npmjs.com/package/redux-simple-router)
-[![npm downloads](https://img.shields.io/npm/dm/redux-simple-router.svg?style=flat-square)](https://www.npmjs.com/package/redux-simple-router)
+**Let react-router do all the work**  :sparkles:
 
-*Let react-router do all the work.*
+[Redux](https://github.com/rackt/redux) is awesome. [React Router](https://github.com/rackt/react-router) is cool. The problem is that react-router manages an important piece of your application state: the URL. If you are using redux, you want your app state to fully represent your UI; if you snapshotted the app state, you should be able to load it up later and see the same thing.
 
-[Redux](https://github.com/rackt/redux) is cool.
-[react-router](https://github.com/rackt/react-router) is neat. The
-problem is that react-router manages an important piece
-of your application state: the URL. If you are using redux, you want
-your app state to fully represent your UI; if you snapshotted the app
-state, you should be able to load it up later and see the same thing.
+react-router does a great job of mapping the current URL to a component tree, and continually does so with any URL changes. This is very useful, but we really want to store this state in redux as well.
 
-react-router automatically maps the current URL to a path down your
-component tree, and continually does so with any URL changes. This is
-very useful, but we really want to store this state in redux as well.
+The entire state that we are interested in boils down to one thing: the URL. This is an extremely simple library that just puts the URL in redux state and keeps it in sync with any react-router changes. Additionally, you can change the URL via redux and react-router will change accordingly.
 
-The entire state that we are interested in boils down to one thing:
-the URL. This is an extremely simple library that just puts the URL in
-redux state and keeps it in sync with any react-router changes.
-Additionally, you can change the URL via redux and react-router will
-change accordingly.
+##### _What about redux-router?_
```

Comment:
> I think it will essentially replace redux-router and that will live as its own standalone project. -- #80 

based on the above, i believe this section will ultimately be removed. maybe adding a deprecation notice (that would be up to acdlite/rackt), but since many people are already using `redux-router` it might not be a bad idea to leave this for now